src/main/webapp/index.html

```diff
@@ -148,8 +147,10 @@ <h1><a href="/">Census Viz</a>
       <div id="data-table"></div>
     </div>
     <footer>
-      Made by Kyra Acquah, Shuli Jones, and Olivia Tobin.
+      <div>Made by Kyra Acquah, Shuli Jones, and Olivia Tobin.</div>
+      <div id="sign-out-btn"><a href="/" onclick="signOut();">Sign out</a></div>
```

Comment:
nit: maybe hide the sign out button when the user is already signed out?

Reply:
I tried to do this this morning but it didn't seem like I was able to modify the style/display of the sign in button.